### PR TITLE
Renames Space Law to Station Law

### PR DESCRIPTION
**Pull request by PopeDaveThe3th**
## What Does This PR Do
Renames Space Law to Station Law. Updates descriptions across many items to reflect this.

## Why It's Good For The Game
THE LORE TEAM HAS SPOKEN.

In all seriousness, the name is misleading, since only Nanotrasen uses this law code.

Additionally, just tossing the word "space" in front of something is both midly immersion-breaking and just not very funny.

## Testing
N/A, just text changes.

## Changelog
:cl:
tweak: Renames Space Law to Station Law
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/paradise_pop.dm [hunk 1]
```diff
@@ -101,7 +101,7 @@
 									"SECRET TECHNIQUE: TABLE AND DISPOSAL!",
 									// Borers got removed but the below reference stays because its hilarious
 									"[pick("MY BROTHER", " MY DOG", "MY BEST FRIEND", "THE BORER", "GEORGE MELONS", "BADMINS")] DID IT!",
-									"; WHATS SPACE LAW?!",
+									"; WHATS CORP LAW?!",
```
Comment (GDNgit) [inline code/modules/reagents/chemistry/reagents/paradise_pop.dm:104]: I think the previous brain damage line was funnier tbh. Corp law just sounds awkward here

Comment (PopeDaveThe3th) [reply code/modules/reagents/chemistry/reagents/paradise_pop.dm:104]: Fair enough/

**Comment by Coolrune206** [comment]
Now I understand **why**, but Space Law is _iconic_.

**Comment by McRamon** [comment]
i dont really understand why, i mean i kinda do but i dont like the vibe of it

**Comment by matttheficus** [comment]
Please please please tell me you got approval from the Design Team for this kinda thing (as well as the Heads of Staff since this touches Space Law and SoP)

**Comment by SynthTwo** [comment]
>     * Wiki (everywhere)
Thinking about what I have to change on the wiki makes me wanna puke.

**Comment by Contrabang** [comment]
Tagging this as "On Hold" since no communication to the design or any of the dev team happened for this. This will need changing of behind the scene stuff like https://paradisestation.org/rules/ and the https://paradisestation.org/wiki/index.php?title=Space_Law, and presumably more. Both of these require behind the scenes changes which should have been commuicated with the development team beforehand.

**Comment by PopeDaveThe3th** [comment]
> Tagging this as "On Hold" since no communication to the design or any of the dev team happened for this. This will need changing of behind the scene stuff like https://paradisestation.org/rules/ and the https://paradisestation.org/wiki/index.php?title=Space_Law, and presumably more. Both of these require behind the scenes changes which should have been commuicated with the development team beforehand.

Ah shit. I'll be perfectly honest I didn't realize that changing some names around would be that big a thing. Apologies.

**Comment by 1080pCat** [comment]
Honestly I'd rather "Corporate law" than station law, since the laws aren't station to station unique, they're laws of the NT corporation.

**Comment by LiberatedWaveMan** [comment]
it gets thrown around as a buzzword often, but yeah, soul removal, and this doesn't come for.. really much of a substantial reason? sure it sounds weird but like, so does a bunch of other stuff in this game, and it really just isn't important enough to warrant overturning a long-running, nearly universal "cultural" piece(?) of sorts. this and the fact that yeah, this would require a metric ton of backend changes- all in all, i just don't think it's important enough for the overhaul and upheaval that it would require

**Comment by Code-Lyoko0** [comment]
This absolutely should not be changed, at least not without the player base at large being okay with it. Space Law is perfectly fine of a name, and renaming something so core to the game just for the sake of changing it would be a terrible choice. Removing the clown planet was fine, but something major like this needs to have wider community approval and not just be the choice of the lore team.

**Comment by Norstead** [comment]
I think throwing the word space in front of something is always funny.

**Comment by PopeDaveThe3th** [comment]
Alright, due to the very negative reviews, ongoing debate over whether corporate or station law is better, and minor kerfuffle regarding just how big of a change this would be, I'll be closing this.

I'll do some far less controversial lore additions in the near future.

**Comment by efzapa** [comment]
late to the party but "Station Law" implies either NT owns one entire station, or they have different lawsets for different stations

both are dumb

**Pull request closed without merging**